src/server/Listener.js

```diff
@@ -151,4 +157,12 @@ function createServer (port, app) {
 	return server;
 }
 
+function createMock (uri) {
+	let keys = []; // eslint-disable-line prefer-const
+	return {
+		keys: keys,
```

Comment:
I'm wondering the same.
The structure of `mocks` is:
```js
mocks = {
	'my/funny/url': {
		get: {
			options: {...},
			clients: [...],
			chunks: [...],
			handler: () => {}
		},
		post: { ... // like get },
		// ... other http verbs
		keys: [], // this never seem to change
		regexp: pathRegexp('my/funny/url', [])
	}
}
```

If somebody sets up a mock with method `keys` or method `regexp` these properties will be overridden. So it seems to be the wrong place to put these.

Maybe it would make more sense to add them like:
```js
mocks = {
	'my/funny/url': {
		get: {
			options: {...},
			clients: [...],
			chunks: [...],
			handler: () => {},
			keys: [], // this never seem to change
			regexp: pathRegexp('my/funny/url', [])
		},
		post: { ... // like get },
		// ... other http verbs
	}
}
```